fosspay/app.py

```diff
@@ -89,3 +95,47 @@ def inject():
         'str': str,
         'int': int
     }
+
+# STRIPE CODE
+@app.route('/support/payment_intents', methods=['POST'])
+def make_payment_intent():
+    # Creates a new PaymentIntent with items from the cart.
+    data = json.loads(request.data)
+    try:
+        amount = Inventory.calculate_payment_amount(items=data['items']) or 500
+        currency = data['currency']
+        # START TODO refactor this code
+        payment_methods = _cfg('payment-methods').split(', ')
+        while("" in payment_methods) : 
+            payment_methods.remove("") 
+        if len(payment_methods) == 0 :
+            payment_methods = ['card']
+        # END TODO 
+        payment_intent = stripe.PaymentIntent.create(
+            amount = amount,
+            currency = currency,
+            payment_method_types = payment_methods
+        )
+        return jsonify({'paymentIntent': payment_intent})
+    except Exception as e:
+        return jsonify(e), 403
+
+@app.route('/payment_intents/<string:id>/shipping_change', methods=['POST'])
```

Comment:
please make sure that the client code doesn't make calls to this route and remove this route. The sponsorship doesn't require shipping.